projects/storefrontstyles/scss/components/myaccount/order-history/_amend-order-items.scss

```diff
@@ -46,16 +32,20 @@
     border-width: 0 0 1px 0;
     border-style: solid;
     border-color: var(--cx-color-light);
-
-    cx-media {
-      text-align: center;
-    }
   }
 
   .cx-info-container {
     @include media-breakpoint-down(sm) {
       display: flex;
       flex-direction: column;
+
+      & > div {
```

Comment:
If possible, please omit creating CSS selectors with `>`, because it makes extensibility harder. Even small change to the HTML structure can cause CSS problems. Ideally, use class here.

Reply:
After some thinking, I've decided to remove this margin, as it seemed unnecessary on second thought.